src/main/java/robot/subsystems/turret/commands/TurnTurret.java

```diff
@@ -0,0 +1,49 @@
+package robot.subsystems.turret.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import robot.subsystems.turret.Turret.Direction;
+
+import static robot.Robot.turret;
+import static robot.Constants.Turret.*;
+
+/**
+ *
+ */
+public class TurnTurret extends Command {
+
+    private Direction direction;
+    private double angle;
+
+    public TurnTurret(double angle) {
+        this(angle, Direction.LEFT);
+    }
+
+    public TurnTurret(double angle, Direction direction) {
+        requires(turret);
+        this.angle = angle;
+        this.direction = direction;
+    }

```

Comment:
The direction choosing should be automatic, calculating which way would be shorter to travel (with some constrains, as your turret cannot rotate indefinitely).